src/main/java/com/minecolonies/coremod/items/ModItems.java

```diff
@@ -1,21 +1,23 @@
 package com.minecolonies.coremod.items;
 
 import net.minecraft.item.Item;
+import net.minecraftforge.registries.IForgeRegistry;
 
 /**
  * Class handling the registering of the mod items.
  */
+@SuppressWarnings("squid:S1444")
 public final class ModItems
 {
-    public static final Item supplyChest  = new ItemSupplyChestDeployer();
-    public static final Item buildTool    = new ItemBuildTool();
-    public static final Item scanTool     = new ItemScanTool();
-    public static final Item permTool     = new ItemScepterPermission();
-    public static final Item caliper      = new ItemCaliper();
-    public static final Item scepterGuard = new ItemScepterGuard();
-    public static final Item supplyCamp   = new ItemSupplyCampDeployer();
-    public static final Item ancientTome  = new ItemAncientTome();
-    public static final Item chiefSword   = new ItemChiefSword();
+    public static Item supplyChest;
+    public static Item buildTool;
+    public static Item scanTool;
+    public static Item permTool;
+    public static Item caliper;
+    public static Item scepterGuard;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make scepterGuard a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)